packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -3267,7 +3267,7 @@ describe('ReactDOMFizzServer', () => {
     });
 
     // Partially render A, but yield before the render has finished
-    await waitFor(['Oops!', 'Oops!']);
+    await waitFor(['Oops!']);
```

Comment:
This one is weird because it used to log twice in both DEV and prod for two different reasons.

In DEV it used to log twice because we did the replay trick which I now removed.

In prod it used to log twice because the component stack generation runs the function again and we don't disable Scheduler.log in prod during this generation - only in dev. To avoid this scenario I use props destructuring so it errors earlier instead.